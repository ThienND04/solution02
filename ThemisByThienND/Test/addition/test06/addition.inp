10
8569 7181 2467 888888587
7441 3410 1529 888888188
6890 7401 1623 888888719
8163 7328 2621 888888164
6247 5435 2284 888888298
7866 7191 2583 888888254
8080 3803 2380 888888963
5615 6115 1497 888888272
8324 7427 2102 888888216
8002 4057 2643 888888165
