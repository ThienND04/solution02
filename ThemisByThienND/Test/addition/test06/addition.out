2602494
10182350
4773583
2519829
5094528
0
5402430
7686068
3040175
4604376
